go/stellar/stellarsvc/frontend.go

```diff
@@ -55,13 +59,89 @@ func (s *Server) GetWalletAccountsLocal(ctx context.Context, sessionID int) (acc
 	return accts, nil
 }
 
+func (s *Server) GetAccountAssetsLocal(ctx context.Context, arg stellar1.GetAccountAssetsLocalArg) (assets []stellar1.AccountAssetLocal, err error) {
+	details, err := s.remoter.Details(ctx, arg.AccountID)
+	if err != nil {
+		s.G().Log.CDebugf(ctx, "remote.Details failed for %q: %s", arg.AccountID, err)
+		return nil, err
+	}
+
+	if len(details.Balances) == 0 {
+		// add an empty xlm balance
+		details.Balances = []stellar1.Balance{
+			stellar1.Balance{
+				Amount: "0",
+				Asset:  stellar1.Asset{Type: "native"},
+			},
+		}
+	}
+
+	displayCurrency, err := s.remoter.GetAccountDisplayCurrency(ctx, arg.AccountID)
+	if err != nil {
+		return nil, err
+	}
+	if displayCurrency == "" {
+		displayCurrency = defaultOutsideCurrency
+		s.G().Log.CDebugf(ctx, "Using default display currency %s for account %s", displayCurrency, arg.AccountID)
+	}
+	rate, err := s.remoter.ExchangeRate(ctx, displayCurrency)
+	if err != nil {
+		s.G().Log.CDebugf(ctx, "exchange rate error: %s", err)
+		return nil, err
+	}
+
+	assets = make([]stellar1.AccountAssetLocal, len(details.Balances))
+	for i, d := range details.Balances {
+		fmtAmount, err := stellar.FormatAmount(d.Amount, false)
+		if err != nil {
+			s.G().Log.CDebugf(ctx, "FormatAmount error: %s", err)
+			return nil, err
+		}
+		asset := stellar1.AccountAssetLocal{
+			Name:         d.Asset.Type,
+			BalanceTotal: fmtAmount,
+			AssetCode:    d.Asset.Code,
+			Issuer:       d.Asset.Issuer,
```

Comment:
For assets this will be a key like `GDVDSIDVX63LKSUFTO5NQ3YHPDDAYUSEPOJLFXVWCVCLEKMOERGHZ7DR`. Is that what frontend wants?

Reply:
We're not supporting anything but XLM at this point, so I was just filling in fields.  I could just remove everything except native for now.

Reply:
For our next milestone, if you import a wallet with non-native currencies what do you see?

Reply:
Just your XLM balance.  We can support more in the future after we get that working completely.

Reply:
I'll filter out anything except native.

Reply:
"we shouldn't launch with them missing" - I got coyne to say that

I'm ok with this PR filtering them out if there's a ticket or something making that happen

Reply:
Ok, my worry is that frontend will spend ages implementing HUGZ and KEYZ asset UIs right now.  I'll make a ticket for opening it up later.